server/schedulers/balance_test.go

```diff
@@ -891,28 +862,21 @@ func checkReplica3(re *require.Assertions, tc *mockcluster.Cluster, sb schedule.
 
 func TestBalanceRegionReplicas5(t *testing.T) {
 	re := require.New(t)
-	ctx, cancel := context.WithCancel(context.Background())
-	defer cancel()
-
-	opt := config.NewTestOptions()
 	// TODO: enable placement rules
-	opt.SetPlacementRuleEnabled(false)
-	tc := mockcluster.NewCluster(ctx, opt)
+	// checkReplica5(re, true)
```

Comment:
Why remove it?

Reply:
it will recover after enabling placement rules, which I will do in another pr.

Reply:
let's repair here in this PR and we can leave the other TODO for now.

Reply:
I think it will be better to split another prhttps://github.com/tikv/pd/commit/3e5da71c510051c37b86e2b41057372ae65061fa

Reply:
#5960